src/store/reducers/devicesReducer.js

```diff
@@ -82,6 +82,21 @@ const insertDeviceReducer = (state, { payload }) => {
   });
 };
 
+const updateTagsReducer = (state, { payload }) => {
+  const updatedTagData = {};
+  payload.commonTags.forEach(({ name, value }) => (updatedTagData[name] = value));
```

Comment:
First, you want me to change map to forEach.  Now, change to reduce. I'm confused as to when to use what. Seems like none of them are "wrong" and they all would work.

Reply:
Nuances of javascript syntax being what they are.  I cannot seem to get reduce to work without also using immutability helper.  Since that seems like overkill and unnecessary overhead in this situation, I am leaving the forEach.